README.md

```diff
@@ -1,31 +1,47 @@
-[![Publish To NPM](https://github.com/palmetto/palmetto-components/workflows/Publish%20To%20NPM/badge.svg)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3A%22Publish+To+NPM%22)
-[![Chromatic](https://github.com/palmetto/palmetto-components/workflows/Chromatic/badge.svg?branch=master)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3AChromatic)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/e61e897623b87d91d155/test_coverage)](https://codeclimate.com/github/palmetto/palmetto-components/test_coverage)
+<p align="center">
+  <a href="https://palmetto.com" rel="noopener" target="_blank"><img width="150" src="https://alchemy.palmetto.com/public//default_logo/1541461188.png" alt="Palmetto logo"></a></p>
+</p>
 
-# Palmetto Components
-A React component library to power all Palmetto UI.
+<h1 align="center" style="border-bottom: none;">Palmetto Component Library</h1>
+<h3 align="center">
+
+  A [React](https://reactjs.org/) component library to power all Palmetto UI
+
+</h3>
+
+<div align="center" style="margin-top: 25px">
+
+  [![Publish To NPM](https://github.com/palmetto/palmetto-components/workflows/Publish%20To%20NPM/badge.svg)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3A%22Publish+To+NPM%22) 
+  [![Chromatic](https://github.com/palmetto/palmetto-components/workflows/Chromatic/badge.svg?branch=master)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3AChromatic) 
+  [![Test Coverage](https://api.codeclimate.com/v1/badges/e61e897623b87d91d155/test_coverage)](https://codeclimate.com/github/palmetto/palmetto-components/test_coverage)
+
+</div>
 
 ## Quick Start
 
 ### 1. Install
+
 `yarn add @palmetto/palmetto-components`
 
-### 2. import global scss/css
+### 2. import global css
+
 ```
-/* main.css or main.scss */
-@import '~@palmetto/palmetto-components/dist/scss/utilities.scss';
-@import '~@palmetto/palmetto-components/dist/scss/variables.scss';
+import '@palmetto/palmetto-components/dist/css/utilities.css';
+import '@palmetto/palmetto-components/dist/css/variables.css';
 ```
 
 ### 3. Use in React component
+
 `import { TextInput } from '@palmetto/palmetto-components'`
 
 
 ## Documentation / Storybook
-[SEE FULL DOCS HERE](https://5ed9214b642dc10022b50a2d-uibkdayawa.chromatic.com/?path=/story/components-button--button)
+
+[SEE FULL DOCS HERE](https://master--5ed9214b642dc10022b50a2d.chromatic.com)
 
 ## Contributing
-Issues and PRs welcome! See [HERE](https://github.com/palmetto/palmetto-components/blob/develop/.github/CONTRIBUTING.md) for contribution guide.
+
+Issues and PRs welcome! See [HERE](https://github.com/palmetto/palmetto-components/blob/master/docs/CONTRIBUTING.md) for contribution guide.
```

Comment:
This needs to be hardcoded to master. Link won't work until this branch gets merged.

As opposed to keeping the docs in `/.github` which is a dir for configuring the inner workings of GitHub, moved this into a `/docs` dir so that it doesn't pollute the route dir with multiple docs, leaving `README.md` as the single entry point.